lte/gateway/fabfile.py

```diff
@@ -544,7 +543,7 @@ def _start_gateway_containerized():
 
     with cd(AGW_ROOT + "/docker"):
         # The `docker-compose up` times are machine dependent, such that a retry is needed here for resilience.
-        run_with_retry('DOCKER_REGISTRY=%s docker-compose -f docker-compose.yaml up -d --quiet-pull' % (env.DOCKER_REGISTRY))
+        run_with_retry(f'./run.py --docker_registry {env.DOCKER_REGISTRY}')
```

Comment:
* Afaiks, this solution makes it impossible to easily add other docker-compose options out of the box, but they will need to be (unnecessarily) re-implemented in the Python wrapper.
* The specification of the `docker-compose.yaml` was dropped, however, it is needed!

Reply:
Okay, so the current way excludes the overwrite file.